bridge/client/app/_services/data.service.ts

```diff
@@ -650,12 +662,20 @@ export class DataService {
       .pipe(map((taskNames) => taskNames.sort((taskA, taskB) => taskA.localeCompare(taskB))));
   }
 
+  public getStageNames(projectName: string): Observable<string[]> {
```

Comment:
I think it is not rocket science to implement this. I think we can remove it. If we need it because we move the component or any new feature needs it I think we can re-implement it.
Also removed the parts in server/../data-service and api-service